src/marionnette_extensions/page.coffee

```diff
@@ -7,10 +7,11 @@ class Page extends Marionette.LayoutView
   canGoBack: ->
     @_parent.canGoBack()
 
-  transitionBack: (route, options) ->
+  transitionBack: (route, options = {}) ->
```

Comment:
I don't think these changes are required. `AnimatableRegion` already has a notion of falling back to the default transition if no transition is specified. (See [here](https://github.com/kabisa/maji/pull/101/files#diff-3f77d8491f3437f4939ba4eb068d0e1bL44)).